safe/definitions/default_settings.py

```diff
@@ -15,6 +15,7 @@
     'showOrganisationLogoInDockFlag': False,
     'developer_mode': False,
     'generate_report': True,
+    'memory_profile': False,
```

Comment:
It will be the default value, if the key doesn't exist. If the user Accept the options dialog, it will be stored in the QSettings. So, more or less, yes.